web/base_pdf_page_view.js

```diff
@@ -166,12 +166,7 @@ class BasePDFPageView {
       }
     };
 
-    const ctx = canvas.getContext("2d", {
-      alpha: false,
-      willReadFrequently: !this.#enableHWA,
```

Comment:
Is `#enableHWA` on this class still used for anything ,or can we completely remove it?